### PR TITLE
Color Redshift DESCQA Test

### PR DESCRIPTION
**Pull request by dkorytov**
This is the initial PR for the color-redshift descqa test. There are couple things that are not quite behaving correctly at the moment though.

### CODE REVIEW

**Comment by dkorytov** [comment]
@yymao, could you review this PR--it's ready. I don't see an option to request a code review now (I didn't request one when I submitted it initially)

**Comment by dkorytov** [comment]
No rush. Thanks!

**Comment by dkorytov** [comment]
@yymao, it would be good to get this branch into master. I'm not sure why the it has failed the travis-ci test.

**Comment by evevkovacs** [comment]
Examples of this test are here: [example 1](https://portal.nersc.gov/project/lsst/descqa/v2/?run=2019-07-16_9&catalog=cosmoDC2_v1.1.4_small&right=2019-07/2019-07-16_9/color_z/cosmoDC2_v1.1.4_small/plot_3.png), [example2](https://portal.nersc.gov/project/lsst/descqa/v2/?run=2019-07-16_7&test=color_z_red_sequence&right=2019-07/2019-07-16_7/color_z_red_sequence/cosmoDC2_v1.1.4_small/plot_2.png), [example3](https://portal.nersc.gov/project/lsst/descqa/v2/?run=2019-07-16_11&test=color_z_red_sequence&right=2019-07/2019-07-16_11/color_z_red_sequence/cosmoDC2_v1.1.4_small/plot_2.png)

File: descqa/ColorRedshiftTest.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+import os
+import numpy as np
+import matplotlib.colors as clr
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+
+
+__all__ = ['ColorRedshiftTest']
+class _CatalogDoesNotHaveQuantity(Exception):
+        """Raised when the catalog doesn't have a quantity and indicates the
```
Comment (evevkovacs) [inline descqa/ColorRedshiftTest.py]: descqa/ColorRedshiftTest.py:11:0: W0311: Bad indentation. Found 8 spaces, expected 4 (bad-indentation)

File: descqa/ColorRedshiftTest.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        test should be skipped"""
```
Comment (evevkovacs) [inline descqa/ColorRedshiftTest.py]: descqa/ColorRedshiftTest.py:13:0: W0311: Bad indentation. Found 8 spaces, expected 4 (bad-indentation)

**Comment by evevkovacs** [comment]
@dkorytov Please click on the details link above in the "continuous-integration/travis-ci/pr " box to see the issues. I added a comments to my code review to indicate which needed to be fixed. Only the warnings are fatal. The bulk of the comments are stylistic and optional. Please fix the warnings and commit the code. Travis runs automatically. If all the warnings are cleared (W021 etc) it should pass. Let me know if you have any questions. The warnings are easy to fix and should only take a few minutes. If you could get this done before Friday, that would be great. Thanks

**Comment by erykoff** [comment]
Just seeing this.  If there were a default set of plots done for halo centrals (>3e13 or so) that is also a great thing to inspect.

**Comment by dkorytov** [comment]
@erykoff , There is already a config for M_halo>1e13 and red sequence galaxies (not centrals). But the a test config file can set up for that quite simply. Let me add it now :)

**Comment by dkorytov** [comment]
Here's how the test looks like: https://portal.nersc.gov/project/lsst/descqa/v2/?run=2019-07-18&test=color_z_massive_centrals

**Pull request merged**